frontend/src/components/EquipmentForm.js

```diff
@@ -1,5 +1,5 @@
-import {useState} from "react"
-import {useEquipmentContext} from "../hooks/useEquipmentContext";
+import { useState } from "react"
+import { useEquipmentContext } from "../hooks/useEquipmentContext";
```

Comment:
is that ";" necessery? Haven't seen it in different files

Reply:
The semicolon is unnecessary. We should avoid using it if not needed.